internal/wazeroir/operations.go

```diff
@@ -118,7 +118,7 @@ type Operation interface {
 	Kind() OperationKind
 }
 
-type OperationKind byte
+type OperationKind uint16
```

Comment:
not really concerned, but any reason why uint16 here? more ops on the way?

Reply:
at wazeroir layer all the instructions are flattened including the MiscOps ones so we would exceed 256 operations :D